src/activitypub/fediverse-bridge.ts

```diff
@@ -55,10 +58,19 @@ export class FediverseBridge {
         const ctx = this.fedifyContextFactory.getFedifyContext();
         let fedifyObject: FedifyNote | Article;
 
+        let mentions: Mention[] = [];
+        let ccs: URL[] = [];
+
         if (post.type === PostType.Note) {
             if (post.inReplyTo) {
                 return;
             }
+            mentions = await this.postService.getMentionsForPost(post);
```

Comment:
What about using `post.mentions` here? Having a `PostService#getMentionsForPost` doesn't seem right to me - maybe if it were a `MentionsService`...

But a mention does seem kinda core to a post at this point, especially as it could potentially trigger notifications/emails/etc... 

I would think something like `post.mentions: Account[]` or `post.mentions: Account["id"][]` would keep everything in the domain, and reduce the number of methods we need on the repository and services.

Reply:
Done [here](https://github.com/TryGhost/ActivityPub/pull/671/files#diff-3522562e572113f90d24c5500be2ae5bf5647a64867dfb920f6727a32a18facfR100)